src/CreateTabs.js

```diff
@@ -0,0 +1,42 @@
+import Tabs from 'react-bootstrap/Tabs'
+import Tab from 'react-bootstrap/Tab'
+import { useState } from 'react';
+
+const CreateTabs = () => {
+  const [key, setKey] = useState('home');
+
+  return (
+    <Tabs
+      id="controlled-tab-example"
+      activeKey={key}
+      onSelect={setKey}
+      className="mb-3"
+    >
+      <Tab eventKey="home" title="Home">
+        Hello!
+      </Tab>
+      
+      <Tab eventKey="profile" title="Profile">
+        Insert pictures and link to podcast
+      </Tab>
+      
+      <Tab eventKey="projects" title="Projects">
+        Insert Projects with descriptions:
+      </Tab>
+      
+      <Tab eventKey="contact" title="Contact">
+        <div>
+          Email:   echoalice0@gmail.com
+        </div><br></br>
+        <div>
+          Twitter:  <a href="https://twitter.com/echo__alice">https://twitter.com/echo__alice</a>
+        </div><br></br>
+        <div>
+          Github:  <a href="https://github.com/EchoAlice">https://github.com/EchoAlice</a>
+        </div>

```

Comment:
Haha okay I'm okay with you doing this in this PR, but you really want to make sure that you're being more clear with your commits. Having the same commit name "Create tabs for the website" doesn't help me figure out what you're doing. When you click on the "Commits" tab of this repo, it should read like a story what you did. So I would imagine to see something like:

1. Create tabs for the website
2. Add Contact info in the Contacts tab
3. Remove unused CSS files

Or something to that extent. We can have a longer talk about git hygiene and I can try to write something down for you. This will be a much longer discussion, especially since it requires a good grasp of git to do this well, but for now, just know that you want all of your changes to be clear in your commits. If all of your commits say the same thing, that's not helping me read it, you know?

Reply:
Ahhhhh
I was thinking that an update to the same commit shouldn't have a different commit comment.  But I see what you're saying.  Definitely makes sense